samples/drivers/adc/adc_dt/sample.yaml

```diff
@@ -33,6 +33,7 @@ tests:
       - longan_nano/gd32vf103/lite
       - rd_rw612_bga
       - frdm_mcxn947/mcxn947/cpu0
+      - ucans32k1sic_s32k146
```

Comment:
```suggestion
      - ucans32k1sic
```

Reply:
pls update this one, otherwise twister will not detect the board

Reply:
Hi, it seems that Twister isn't detecting the board in either case:

ERROR   - samples/drivers/adc/adc_dt/sample.drivers.adc.adc_dt - platform_allow - unrecognized platform - ucans32k1sic/s32k146

ERROR   - samples/drivers/adc/adc_dt/sample.drivers.adc.adc_dt - platform_allow - unrecognized platform - ucans32k1sic_s32k146

Reply:
ok, use `ucans32k1sic` then. That's what I've been using in other tests/samples

Reply:
ok, use `ucans32k1sic` then. That's what I've been using in other tests/samples